pkg/storage/fs/eos/eos.go

```diff
@@ -846,64 +846,16 @@ func (fs *eosfs) createShadowHome(ctx context.Context) error {
 		return errors.Wrap(err, "eos: error verifying if user home directory exists")
 	}
 
-	// TODO(labkode): only trigger creation on not found, copy from CERNBox logic.
-	err = fs.c.CreateDir(ctx, "root", home)
```

Comment:
AFAIU using `root` as the owner is by design. We don't want users that use eos directly to mess with the shadow tree. It is managed exclusively by reva.

reva will manage the tree using root / elevated permissions

@labkode correct me if I am wrong

Reply:
@butonic you're right, this directory is owned by root, not by the user.
Accessing the shadow tree is possible, your EOS instances are not correctly configured, that is why you cannot access the shadow tree. The machine/box where Reva runs needs to be configured:
- as a trusted unix gateway with sudo permissions
- using keytab with sudo permissions